docs/design/README.md

```diff
@@ -1,50 +1,34 @@
-- [Proposal: Maintaining histograms in plan](./2018-09-04-histograms-in-plan.md)
-- [Proposal: Support a Global Column Pool](./2018-10-22-the-column-pool.md)
-- [Proposal: Join Reorder Design v1](./2018-10-20-join-reorder-dp-v1.md)
-- [Proposal: Support Window Functions](./2018-10-31-window-functions.md)
-- [Proposal: Access a table using multiple indexes](./2019-04-11-indexmerge.md)
-- [Proposal: Collations in TiDB](./2020-01-24-collations.md)
-
-### Completed
-
-- [Proposal: A new aggregate function execution framework](./2018-07-01-refactor-aggregate-framework.md)
-- [Proposal: TiDB DDL architecture](./2018-10-08-online-DDL.md)
-- [Proposal: Infer the System Timezone of a TiDB cluster via TZ environment variable](./2018-09-10-adding-tz-env.md)
-- [Proposal: Table Partition](./2018-10-19-table-partition.md)
-- [Proposal: Implement View Feature](./2018-10-24-view-support.md)
-- [Proposal: Support restoring SQL text from an AST tree](./2018-11-29-ast-to-sql-text.md)
-- [Proposal: Support Plugin](./2018-12-10-plugin-framework.md)
-- [Proposal: Support Skyline Pruning](./2019-01-25-skyline-pruning.md)
-- [Proposal: Support Index Merge](./2019-04-11-indexmerge.md)
-- [Proposal: Support Automatically Index Recommendation](./2019-11-05-index-advisor.md)
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the TiDB community.
+
+The process described in this page is intended to provide a consistent and controlled path for new features to enter the TiDB projects, so that all stakeholders can be confident about the direction the projects is evolving in.
+
+## Who should initiate the design document?
+
+Anyone can initiate a design document but you shouldn't do it unless you have an intention of getting the work done to implement it.
+
+## Before creating a design document
+
+A hastily-proposed design document can hurt its chances of acceptance. Low quality proposals, proposals for previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of the design document can make the process smoother.
+
+Although there is no single way to prepare for submitting a design document, it is generally a good idea to pursue feedback from other project developers beforehand, to ascertain that the design document may be desirable; having a consistent impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an design document for now is [creating a disucss issue](https://github.com/pingcap/tidb/issues/new/choose), which is going to be converted into a tracking issue of the design implementation.
```

Comment:
We should leverage GitHub Discussion / Mailing List / Forum / Slack later when we have an active and clear communication way. I can image the discuss issue would quickly vanish from the first page of our issue view. But we don't have a better open way yet.